### PR TITLE
Created keras version of backjack_dqn and dqn_agent

### PR DESCRIPTION
**Pull request by YileAllenChen1**
Created keras version of backjack_dqn and dqn_agent

### CODE REVIEW

File: examples/blackjack_dqn_keras.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+episode_num = 100000
+
+# The intial memory size
+memory_init_size = 100
+
+# Train the agent every X steps
+train_every = 1
+
+# The paths for saving the logs and learning curves
+log_dir = './experiments/blackjack_dqn_keras_result/'
```
Comment (ThaFurther) [inline examples/blackjack_dqn_keras.py:30]: Probably want to use `os.path` module.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# RLCard: A Toolkit for Reinforcement Learning in Card Games
+# RLCard: A Toolkit for Reinforcement Learning in Card GameS
```
Comment (Michael1015198808) [inline README.md:1]: Why use capital <kbd>s</kbd>?

File: examples/blackjack_dqn_keras.py (hunk 1 above)
Comment (YileAllenChen1) [reply examples/blackjack_dqn_keras.py:30]: I referenced the format of the other examples, for line 30 they all wrote it this way. Do you want to change all the other examples to os.path as well?

File: README.md (hunk 2 above)
Comment (YileAllenChen1) [reply README.md:1]: Sorry, where is this README.md located? I did not encounter it.

**Pull request closed without merging**